opsmngr/automation_config.go

```diff
@@ -71,8 +72,28 @@ func (s *AutomationConfigServiceOp) Update(ctx context.Context, groupID string,
 	return resp, err
 }
 
+// See more: https://docs.cloudmanager.mongodb.com/reference/api/automation-config/#update-the-automation-configuration
+func (s *AutomationConfigServiceOp) UpdateAgent(ctx context.Context, groupID string) (*AutomationConfigAgent, *atlas.Response, error) {
+	basePath := fmt.Sprintf(automationConfigBasePath+"/updateAgentVersions", groupID)
```

Comment:
can you please add here
```suggestion
	if groupID == "" {
		return nil, nil, atlas.NewArgError("groupID", "must be set")
	}
	basePath := fmt.Sprintf(automationConfigBasePath+"/updateAgentVersions", groupID)
```